v2/emailpassword/custom-ui/enable-email-verification.mdx

```diff
@@ -838,7 +838,7 @@ Once the user has enters their email, you can call the following API to send a r
 ```bash
 curl --location --request POST '^{form_apiDomain}^{form_apiBasePath}/user/email/verify/token' \
 --header 'rid: emailverification' \
---header 'Cookie: sAccessToken=...; sIdRefreshToken=...'
+--header 'Cookie: sAccessToken=...'
```

Comment:
So for mobile apps, we will default to header based. Therefore this (and other curl commands like this one) should use the authorization bearer token